plugins/rbac/src/utils/rbac-utils.ts

```diff
@@ -180,6 +180,7 @@ export const getPermissionsData = (
               policy: policyTitleCase || 'Use',
               effect: policy.effect,
             }),
+            isResourced: true,
```

Comment:
@debsmita1 this value should come from `RoleBasedPolicy`. 
```suggestion
            isResourced: policy.isResourced,
```

Reply:
Thank you for bringing this to my attention. I noticed the isResourced property in the permission policies API response while addressing a different RBAC issue.